meta-lmp-base/recipes-core/systemd/systemd_%.bbappend

```diff
@@ -7,7 +7,6 @@ PACKAGECONFIG ?= " \
     ${@bb.utils.filter('MACHINE_FEATURES', 'tpm2', d)} \
     ${@bb.utils.contains('DISTRO_FEATURES', 'wifi', 'rfkill', '', d)} \
     ${@bb.utils.contains('DISTRO_FEATURES', 'x11', 'xkbcommon', '', d)} \
-    ${@bb.utils.contains('DISTRO_FEATURES', 'efi', 'gnu-efi', '', d)} \
```

Comment:
Same, please describe where it was dropped from upstream as part of the commit message.